pymatgen/io/vasp/outputs.py

```diff
@@ -386,14 +386,17 @@ def __init__(self, filename, ionic_step_skip=None,
             if parse_potcar_file:
                 self.update_potcar_spec(parse_potcar_file)
 
-        if self.incar["ALGO"] == "BSE":
+        try:
+            if "BSE" in self.incar["ALGO"]:
```

Comment:
This should not be a try-except. self.incar.get("ALGO") == "BSE" is better.

Reply:
I see.

Reply:
self.incar.get("ALGO", "") == "BSE"